app/browser/api/userModel.js

```diff
@@ -115,11 +115,11 @@ function randomKey (dictionary) {
   return keys[keys.length * Math.random() << 0]
 }
 
-const goAheadAndShowTheAd = (windowId, categoryName, notificationText, notificationUrl) => {
+const goAheadAndShowTheAd = (windowId, advertiser, notificationText, notificationUrl) => {
```

Comment:
shouldn't we just call this `notificationTitle` ?

Reply:
i think from the browser's perspective: it needs a title, text, and url for the notification.

it doesn't really matter if the title was a category yesterday, an advertiser today, or something else tomorrow..